src/others/sysmon/lv_sysmon.c

```diff
@@ -137,6 +138,34 @@ void lv_sysmon_hide_performance(lv_display_t * disp)
     lv_obj_add_flag(disp->perf_label, LV_OBJ_FLAG_HIDDEN);
 }
 
+void lv_sysmon_dump_performance(lv_display_t * disp)
```

Comment:
Let's use simpler names for the API
```suggestion
void lv_sysmon_dump(lv_display_t * disp)
```

Reply:
Maybe we cannot use this simpler name like lv_sysmon_dump, because we only dump the performance infomations, not include the memory usage infomations. If we need dump the memory usage infos i'd like push antoher patch for it. For now, let's concentrate on the performance feature.

Reply:
Makes sense. Thank you :)